common/src/certificate.rs

```diff
@@ -6,4 +6,13 @@ pub struct AddCertificateRequest {
 }
 
 #[derive(Deserialize, Serialize, Debug)]
-pub struct CertificateResponse {}
+pub struct DeleteCertificateRequest {
+    pub domain: String,
+}
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct CertificateResponse {
+    pub subject: String,
+    pub serial_hex: String,
+    pub not_after: String,
+}
```

Comment:
Are these duplicated in ecs-shuttle, or are they used for different things?

Reply:
The req models are similar but different due to these not needing project id